estate/__init__.py

```diff
@@ -0,0 +1 @@
+from . import models
```

Comment:
Try to put an EOL character on the last line of file, otherwise when people will append code to the file, it will show up as impacting the former last line.